doc/pages/FAQ.md

```diff
@@ -62,7 +62,35 @@ remove) are also run in a sandbox and guaranteed not to affect your system.
 
 ---
 
-#### 🐫  Why does ``opam init`` need to add stuff to my init scripts / why is ``eval $(opam env)`` needed ?
+#### 🐫  Why does opam require ``bwrap`` installed ?
```

Comment:
Sorry, I'm on a rampage recently about French spacing of punctuation :wink: No space before the question mark.

I'd delete the word *installed* and just have "Why does opam require `bwrap`?"